docs/sphinx/developers/generating-test-images.txt

```diff
@@ -39,66 +39,109 @@ global metadata map:
     echo "my.key=some.value" >> my-special-test-file.fake.ini
 
 
-There are a few other keys that can be added as well:
+There are several other keys that can be added as well, these are listed below
+with their default values.
 
 .. tabularcolumns:: |p{3cm}|p{11cm}|
 
 .. list-table::
     :header-rows: 1
-    :widths: 30, 70
+    :widths: 30, 70, 10
```

Comment:
I think according to the docs that the `:widths:` are relative, so they can be greater than 100%.  Or you could adjust it to add up to 100%; it doesn't really matter.  http://docutils.sourceforge.net/docs/ref/rst/directives.html#list-table